### PR TITLE
ISRmatching

### PR DESCRIPTION
**Pull request by semrat**
Create a module to the ISR gen matching for identification of the ISR jets

### CODE REVIEW

File: Framework/scripts/getSamplesCfg.sh [hunk 1]
```diff
@@ -3,7 +3,8 @@
 cfgDir=${CMSSW_BASE}/src/Framework/Framework/cfg
 
 #Default options
-sampleSCfgFile="sampleSets_v6.cfg"
+#sampleSCfgFile="sampleSets_v6.cfg"
```
Comment (cmadrid1) [inline Framework/scripts/getSamplesCfg.sh:6]: I don't think we are ready to make this the default yet.

Comment (semrat) [reply Framework/scripts/getSamplesCfg.sh:6]: yeah I just needed for ISR gen matching

Comment (cmadrid1) [reply Framework/scripts/getSamplesCfg.sh:6]: Then I can't merge this PR in with this change.

Comment (semrat) [reply Framework/scripts/getSamplesCfg.sh:6]: Do you want me re do that change to merge? Or you will merge later ?

Comment (cmadrid1) [reply Framework/scripts/getSamplesCfg.sh:6]: It is up to you.

**Pull request merged**